### DISCUSSION TITLE
ReactiveUI Thread Scheduling

### DISCUSSION THREAD

**Discussion by Haale** [reactiveui/ReactiveUI#3850] [General]
when I use vs2017 and ReactiveUI to develop a WinForm app (binding model property to a UI ctronl)， Debug outputs:
```
Thread 0x555c has exited with return value 0 (0x0).
Thread 0x5524 has exited with return value 0 (0x0).
Thread 0x4d24 has exited with return value 0 (0x0).
Thread 0x3dcc has exited with return value 0 (0x0).
Thread 0x29e8 has exited with return value 0 (0x0).
Thread 0x165c has exited with return value 0 (0x0).
Thread 0x5174 has exited with return value 0 (0x0).
Thread 0x286c has exited with return value 0 (0x0).
Thread 0x5fdc has exited with return value 0 (0x0).
Thread 0x3a4c has exited with return value 0 (0x0).
Thread 0x5320 has exited with return value 0 (0x0).
Thread 0x3ba4 has exited with return value 0 (0x0).
Thread 0x8 has exited with return value 0 (0x0).
Thread 0x3110 has exited with return value 0 (0x0).
Thread 0x3d30 has exited with return value 0 (0x0).
Thread 0x4be0 has exited with return value 0 (0x0).
Thread 0x5098 has exited with return value 0 (0x0).
Thread 0xfc has exited with return value 0 (0x0).
Thread 0x2ea4 has exited with return value 0 (0x0).
Thread 0x31a0 has exited with return value 0 (0x0).
Thread 0x1c88 has exited with return value 0 (0x0).
Thread 0x536c has exited with return value 0 (0x0).
Thread 0x5d7c has exited with return value 0 (0x0).
Thread 0xa68 has exited with return value 0 (0x0).
Thread 0x3e1c has exited with return value 0 (0x0).
Thread 0x2618 has exited with return value 0 (0x0).
Thread 0x894 has exited with return value 0 (0x0).

```

Is this normal? if I comment the relevant code（model derived from ReactiveObject initializing and binding）, nothing happened to output. Why?

**Comment by ChrisPulman** [comment]
Which version of ReactiveUI are you using?

A repo showing the issue would help so that I can see which ReactiveUI and third party components you are trying to use.

I would need to locate this old version of Visual Studio to trace the cause, different versions have of ReactiveUI have different levels of Visual Studio support based upon the various components you use.

If your using the latest releases of ReactiveUI I would recommend that you use Visual Studio 2022. Otherwise you may need to find a version with compatible components for Visual Studio 2017, for example V8.0.0, or something in this version range.

**Reply by Haale** [reply]
> Which version of ReactiveUI are you using?
> 
> A repo showing the issue would help so that I can see which ReactiveUI and third party components you are trying to use.
> 
> I would need to locate this old version of Visual Studio to trace the cause, different versions have of ReactiveUI have different levels of Visual Studio support based upon the various components you use.
> 
> If your using the latest releases of ReactiveUI I would recommend that you use Visual Studio 2022. Otherwise you may need to find a version with compatible components for Visual Studio 2017, for example V8.0.0, or something in this version range.

ReactiveUI v20.1.0.0+ ReactiveUI.Winform v20.1.0.0. + .NET Framework 4.8 (latest) + VS2017 (v15.9)+Win10 22H2
And where can I get the list of ReactiveUI compatible with Visual Studio version?
